tools/devops/automation/templates/mac/build.yml

```
@@ -238,3 +239,20 @@ steps:
     SYSTEM_ACCESSTOKEN: $(System.AccessToken)
     MONO_DEBUG: no-gdb-backtrace
   continueOnError: true
+
+- pwsh: |
+    Import-Module $Env:SYSTEM_DEFAULTWORKINGDIRECTORY\xamarin-macios\tools\devops\automation\scripts\GitHub.psm1
+    $testsPath = "$(Build.SourcesDirectory)/artifacts/mac-test-package/tests"
+    if (-not ([IO.File]::Exists("$testsPath/test-run-completed"))) {
+      Set-GitHubStatus -Status "error" -Description "Tests timed out on macOS $Env:CONTEXT." -Context "$Env:CONTEXT"
+      $request = New-GitHubComment -Header "Tests timed out on macOS $Env:CONTEXT" -Description "" -Message "" -Emoji ":x:"
```

```suggestion
      $request = New-GitHubComment -Header "Tests timed out on macOS $Env:CONTEXT"  -Message "" -Emoji ":x:"
```
Don't even pass description since you made the change for that in the PR.